std/zip.d

```diff
@@ -1,61 +1,149 @@
 // Written in the D programming language.
 
 /**
- * Read/write data in the $(LINK2 https://en.wikipedia.org/wiki/Zip_%28file_format%29, zip archive) format.
- * Makes use of the etc.c.zlib compression library.
- *
- * Limitations:
- *      $(UL
- *      $(LI Multi-disk zips not supported.)
- *      $(LI Only Zip version 20 formats are supported.)
- *      $(LI Only supports compression modes 0 (no compression) and 8 (deflate).)
- *      $(LI Does not support encryption.)
- *      )
- *
- * Example:
- * ---
-// Read existing zip file.
-import std.digest.crc, std.file, std.stdio, std.zip;
-
-void main(string[] args)
+Read and write data in the $(LINK2 https://en.wikipedia.org/wiki/Zip_%28file_format%29, zip archive) format.
+
+Standards:
+
+The current implementation mostly conforms to $(LINK2 https://www.iso.org/standard/60101.html, ISO/IEC 21320-1:2015),
```

Comment:
Mostly, because there might be some nonconformation left due to UTF8 issues or with the flags. I'm not totally in this currently, but have to investigate soon for fixing some further bugs.